frontend/src/lib/components/Support/SupportModal.tsx

```diff
@@ -45,6 +45,12 @@ export function SupportModal({ loggedIn = true }: { loggedIn?: boolean }): JSX.E
     const { setSendSupportRequestValue, closeSupportForm } = useActions(supportLogic)
     const { objectStorageAvailable } = useValues(preflightLogic)
 
+    if (!preflightLogic.values.preflight?.cloud) {
+        if (isSupportFormOpen) {
+            lemonToast.error(`Support tickets are only supported on PostHog cloud`)
```

Comment:
We can just hide the whole support section on self-hosted, since we explicitly don't offer support for hobby setups. In terms of UX, there should _never_ be any button in an interface whose only function is showing an error toast

Reply:
Sure, but as a backup I want us to block folks from being able to send requests if we fail to exclude the button in some of the locations.

Reply:
That's fair, but the primary strategy should still be hiding the support buttons/links themselves. This PR doesn't do that.

Reply:
will do a separate PR for that as it could be controversial what exactly we hide